alembic/ddl/postgresql.py

```diff
@@ -650,10 +651,16 @@ def _exclude_constraint(
 
 
 def _render_potential_column(
-    value: Union[ColumnClause, Column], autogen_context: AutogenContext
+    value: Union[ColumnClause, Column, TextClause], autogen_context: AutogenContext
 ) -> str:
-    if isinstance(value, ColumnClause):
-        template = "%(prefix)scolumn(%(name)r)"
+    if isinstance(value, (ColumnClause, TextClause)):
```

Comment:
whatever is being fixed here seems like it should apply to Index also

Reply:
I can look into index to see how it behaves. I think there are tests of expressions and text in alembic with indexes

Reply:
ok, index just renders everything as create_index(..., [sa.text(), sa.text()]) since it never renders it inline it does not have an issue with it. In particular it will convert even function into `text()`.

exclude constraint cannot really do that since it can be rendered inline and until we fixed it it did not handle text at all.

so I think this could be a separate function, but maybe the logic could be moved to rendered.py?